lib/livebook_cli/server.ex

```diff
@@ -58,6 +59,14 @@ defmodule LivebookCLI.Server do
           browser_open(LivebookWeb.Endpoint.access_url())
         end
 
+        if opts[:open_new] do
+          LivebookWeb.Endpoint.access_url()
+          |> URI.parse()
+          |> Map.update!(:path, &(&1 <> "explore/notebooks/new"))
```

Comment:
Isn't the path `nil` for a root URL?

Reply:
nope, it's `""`!

Reply:
I mean, it is `nil` for root URL. Here, it is not `""` but `"/"`, sorry about that. The whole reason we do the URI.parse/URI.to_string dance is because the URI contains the auth token. So really the url we're parsing is `http://localhost:8080/?token=x`.

Reply:
```suggestion
          |> Map.update!(:path, &((&1 || "/") <> "explore/notebooks/new"))
```